index.html

```diff
@@ -2,40 +2,84 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Awesome books</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Awsome Books</title>
   </head>
   <body>
+    <header>
+      <div class="logo"><p>Awesome Books</p></div>
+      <ul class="navbar">
+        <li ><a href="#list" id="list-link" class="links active">List</a></li>
+        <li ><a href="#add_book" id="add-link" class="links">Add new</a></li>
+        <li ><a href="#contact" id="contact-link" class="links">Contact</a></li>
+      </ul>
+    </header>
     <main>
-      <!-- Header Section  -->
-
-      <section id="message-container"></section>
-      <table class="books_table"></table>
-
-      <section id="add_book">
-        <h2>Awesome Books</h2>
-        <div id="add" class="add-new section hidden">
-          <form action="#" method="get" id="book-form">
-            <input
-              type="text"
-              id="title"
-              class="text-inpute"
-              placeholder="Title"
-            /><br />
-            <input
-              type="text"
-              id="author"
-              class="text-inpute"
-              placeholder="Author"
-            /><br />
-            <button type="submit" id="button">Add</button>
-          </form>
-          <span id="error"></span>
+      <!-- Header Section -->
+      
+       <section id="message-container">
+        
+          </section>
+      
+      <!-- List Section -->
+      <section class="books_container" id="list">
+         <h1>Awesome books</h1>
+        <div>
+          <table class="books_table"> 
+           <thead>
+            <tr class="line_hd">
+             <th>Title</th>
+             <th>Author</th>
+             <th>Action</th>
+            </tr>
+          </thead>
+      <!--- List of all the books -->
+        </table> 
         </div>
       </section>
+     
+    <section id="add_book" >
+      <h2>Add a new book</h2>
+      <div id="add" class="add-new section hidden">
+        
+        <form action="#" method="get" id="book-form">
+          
+          <input
+            type="text"
+            id="title"
+            class="text-inpute"
+            placeholder="Title"
+          />
+          <input
+            type="text"
+            id="author"
+            class="text-inpute"
+            placeholder="Author"
+          />
+          <button type="submit" id="button">Add</button>
+        </form>
+        <span id="error"></span>
```

Comment:
- [ ] Good job so far, 👏 but there is still an issue, the form to allow the user to add a book does not display once we click on the `Add new button; the only thing that we can see is the text `Add a new book`.
![Screenshot from 2022-10-25 19-34-10](https://user-images.githubusercontent.com/89228743/197867165-d8534765-f8fd-4810-8447-fd7ab661b141.png)